app/controllers/project1.php

```diff
@@ -0,0 +1,279 @@
+<?php
+
+class ProjectsController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /projects
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{
+		$pTitle		= "Projects";
+
+		$counter 	=	0;
+		$user 		=	User::find(Auth::id());
+		$projects 	=	$user->projects()->get();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "$projects" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.137:9000/coding_rules#rule_key=php%3AS1481)